NuGetPackageVerifier.json

```diff
@@ -14,6 +14,7 @@
     "adx-nonshipping": {
         "rules": [],
         "packages": {
+            "Microsoft.AspNetCore.DiagnosticsViewPage.Sources": { },
```

Comment:
@kichalla added it here: https://github.com/aspnet/specs/commit/c0533ba951992e697f9a9afc8f39102ad13d3bd4